crates/rbuilder/src/building/mod.rs

```diff
@@ -137,7 +134,7 @@ impl BlockBuildingContext {
                 },
             )
             .ok()?;
-        block_env.coinbase = signer.address;
+        evm_env.block_env.coinbase = signer.address;
```

Comment:
Do we need https://github.com/rkrasiuk/rbuilder/commit/d7b1637967dffc1f4f3ade5303aacc185963c5c3 /cc @rkrasiuk